cli/parsefile.go

```diff
@@ -226,7 +226,7 @@ func printPmt(pn uint16, pmt psi.PMT) {
 	printlnf("\tPIDs %v", pmt.Pids())
 	println("\tElementary Streams")
 	for _, es := range pmt.ElementaryStreams() {
-		printlnf("\t\tPid %v : StreamType %v", es.ElementaryPid(), es.StreamType())
+		printlnf("\t\tPid %v: StreamType %v: %v", es.ElementaryPid(), es.StreamType(), es.StreamTypeDescription())
```

Comment:
Should we change from `%v` to `%s` here for `es.StreamTypeDescription()`?

Also I know it's not part of this PR but the others should be `%d`s.

Reply:
Seems easier to just leave them all as `%v` and let the runtime figure it out.